src/embed_tests/TestPythonEngineProperties.cs

```diff
@@ -181,13 +181,42 @@ public void SetProgramName()
         public void SetPythonPath()
         {
             PythonEngine.Initialize();
-            string path = PythonEngine.PythonPath;
+
+            const string moduleName = "pytest";
+            bool importShouldSucceed;
+            try
+            {
+                Py.Import(moduleName);
+                importShouldSucceed = true;
+            }
+            catch
+            {
+                importShouldSucceed = false;
+            }
+
+            dynamic sys = Py.Import("sys");
+            var locals = new PyDict();
+            locals.SetItem("sys", sys);
+            var result = PythonEngine.Eval($"'{System.IO.Path.PathSeparator}'.join(sys.path)", null, locals.Handle);
```

Comment:
NIT: you don't need `locals`. You could do something like
`Py.Import("sys").GetAttr("path").As<string[]>()`. Then use .NET `string.Join`